iib/workers/tasks/build_merge_index_image.py

```diff
@@ -153,31 +155,33 @@ def _add_bundles_missing_in_source(
     )
     _add_label_to_index(
         'com.redhat.index.delivery.distribution_scope',
-        distribution_scope,
+        # MYPY error: Argument 2 to "_add_label_to_index"
```

Comment:
In this case _add_label_to_index method should be updated to tolerate None as label or distribution scope should be changed make distribution_scope non-null

Reply:
Do I understand right that you are proposing those two solutions?
- None value should be accepted as the label_key parameter in the _add_label_to_index method
-  distribution_scope should be set to some empty value when it`s value is None
If I am right, which of that solutions is better according to you? Personally I like second one more, as it does not interfere with other part of the code - in some cases Mypy check that you are not sending None, could be useful.